sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UseValueParameter.cs

```diff
@@ -64,6 +64,15 @@ protected sealed override void Initialize(SonarAnalysisContext context)
                         return;
                     }
 
+                    ISymbol symbol2;

```

Comment:
Perhaps rename to something like "unused"? Symbol2 looks quite weird.

If we could refactor the method not to return the symbol in the first place, it would be even better, but may not be worth it.